includes/organization_config.php

```diff
@@ -11,52 +11,52 @@ function organization_config_mnu() {
-		$org_options['recaptcha_publickey'] = isset($_POST['recaptcha_publickey']) && !empty($_POST['recaptcha_publickey']) ? $_POST['recaptcha_publickey'] : '';
-		$org_options['recaptcha_privatekey'] = isset($_POST['recaptcha_privatekey']) && !empty($_POST['recaptcha_privatekey']) ? $_POST['recaptcha_privatekey'] : '';
-		$org_options['recaptcha_theme'] = isset($_POST['recaptcha_theme']) && !empty($_POST['recaptcha_theme']) ? $_POST['recaptcha_theme'] : '';
-		$org_options['recaptcha_width'] = isset($_POST['recaptcha_width']) && !empty($_POST['recaptcha_width']) ? $_POST['recaptcha_width'] : '';
-		$org_options['recaptcha_language'] = isset($_POST['recaptcha_language']) && !empty($_POST['recaptcha_language']) ? $_POST['recaptcha_language'] : '';
-		$org_options['google_maps_api_key'] = isset($_POST['google_maps_api_key']) && !empty($_POST['google_maps_api_key']) ? $_POST['google_maps_api_key'] : '';
-		$org_options['espresso_dashboard_widget'] = isset($_POST['espresso_dashboard_widget']) && !empty($_POST['espresso_dashboard_widget']) ? $_POST['espresso_dashboard_widget'] : '';
-		$org_options['time_reg_limit'] = isset($_POST['time_reg_limit']) && !empty($_POST['time_reg_limit']) ? $_POST['time_reg_limit'] : '';
-		$org_options['skip_confirmation_page'] = isset($_POST['skip_confirmation_page']) ? $_POST['skip_confirmation_page'] : 'N';
-		$org_options['allow_mer_discounts'] = isset($_POST['allow_mer_discounts']) ? $_POST['allow_mer_discounts'] : 'N';
-		$org_options['allow_mer_vouchers'] = isset($_POST['allow_mer_vouchers']) ? $_POST['allow_mer_vouchers'] : 'N';
-		$org_options['use_attendee_pre_approval'] = isset($_POST['use_attendee_pre_approval']) && !empty($_POST['use_attendee_pre_approval']) ? $_POST['use_attendee_pre_approval'] : '';
+		$org_options['email_before_payment'] = isset($_POST['email_before_payment']) && !empty($_POST['email_before_payment']) ? sanitize_text_field($_POST['email_before_payment']) : '';
+		$org_options['email_fancy_headers'] = isset($_POST['email_fancy_headers']) && !empty($_POST['email_fancy_headers']) ? sanitize_text_field($_POST['email_fancy_headers']) : '';
+		$org_options['use_captcha'] = isset($_POST['use_captcha']) && !empty($_POST['use_captcha']) ? sanitize_text_field($_POST['use_captcha']) : '';
+		$org_options['recaptcha_publickey'] = isset($_POST['recaptcha_publickey']) && !empty($_POST['recaptcha_publickey']) ? sanitize_text_field($_POST['recaptcha_publickey']) : '';
+		$org_options['recaptcha_privatekey'] = isset($_POST['recaptcha_privatekey']) && !empty($_POST['recaptcha_privatekey']) ? sanitize_text_field($_POST['recaptcha_privatekey']) : '';
+		$org_options['recaptcha_theme'] = isset($_POST['recaptcha_theme']) && !empty($_POST['recaptcha_theme']) ? sanitize_text_field($_POST['recaptcha_theme']) : '';
+		$org_options['recaptcha_width'] = isset($_POST['recaptcha_width']) && !empty($_POST['recaptcha_width']) ? sanitize_text_field($_POST['recaptcha_width']) : '';
```

Comment:
seeing how we only expect integers, using
` intval($_POST['recaptcha_width'])` would make sense. But currently the code sets this to a string, so to be consistent with that (just in case changing the value to an int somehow breaks things) we could do
```suggestion
		$org_options['recaptcha_width'] = isset($_POST['recaptcha_width']) && !empty($_POST['recaptcha_width']) ? (string)intval($_POST['recaptcha_width']) : '';
```

Reply:
Thanks